package.json

```diff
@@ -211,6 +212,11 @@
         "command": "extension.rebuildIntelliSenseCache",
         "title": "%extension.rebuildIntelliSenseCache%",
         "category": "MS SQL"
+      },
+      {
+        "command": "mssql.loadCompletionExtension",
+        "title": "%mssql.loadCompletionExtension%",
+        "category": "MS SQL"
```

Comment:
does this show up in the Command Palette?

Reply:
This is just for testing, it's removed now.